README.md

```diff
@@ -1,8 +1,31 @@
 # 🥧 PIE (PHP Installer for Extensions)
 
+## What is PIE?
+
+PIE is a new installer for PHP extensions, intended to eventually replace PECL.
+It is distributed as a [PHAR](https://www.php.net/manual/en/intro.phar.php),
+just like Composer, and works in a similar way to Composer, but it installs PHP
+extensions (PHP Modules or Zend Extensions) to your PHP installation, rather
+than pulling PHP packages into your project or library.
+
+## What do I need to get started?
+
 You will need PHP 8.1 or newer to run PIE, but PIE can install an extension to
 any installed PHP version.
 
+On Linux, you will need a build toolchain installed. On Debian/Ubuntu type
+systems, you could run something like:
+
+```shell
+sudo apt install build-essential autoconf libtool bison re2c pkg-config
```

Comment:
build-essential is a meta package to build Debian packages. It is not correct to list here (it's a common mistake, due to the bad name of the package). 
```suggestion
sudo apt install gcc make autoconf libtool bison re2c pkg-config
```

This should (hopefully) be sufficient to have everything needed to build extensions.